controllers/controllers.error.js

```diff
@@ -0,0 +1,16 @@
+exports.handle404Paths = (req,res,next) => {
```

Comment:
could we extract this out of the controller folder, technically not a controller.